cookbooks/aws-parallelcluster-environment/files/amazon-2023/network_interfaces/configure_nw_interface.sh

```diff
@@ -3,16 +3,27 @@
 set -ex
```

Comment:
thank you for aligning all Os specific scripts.
Since we are doing it, please use the same execution options: `set -ex` everywhere to facilitate troubleshooting in case of issues